src/main/resources/config.yml

```diff
@@ -509,16 +509,30 @@ custom-events:
   #Reward a player a price in your economy currency every time he is still alive after the interval (in seconds)
   time:
     enable: false
+    # Amount of money the player should get
     reward: 100
+    # Enter a list of commands that should be executed instead
```

Comment:
Add ``(Change to commands: [] if you do not want any commands to be executed.)``